src/pages/p/index.tsx

```diff
@@ -1,5 +1,5 @@
-import FeedbackFormButton from 'components/shared/FeedbackFormButton'
```

Comment:
Question: do we need these to be 1:1 mapping to routes (`p`), or could we give this folder a more meaningful name?

Reply:
Also, in this PR, could we take the chance to move code out of index files and have them as pure redirects for files?

Reply:
@wraeth-eth  yup, needs to be a 1:1 mapping.

Next routes are built from the file tree.

It also uses `index` files to determine routes. `pages/index.ts` is the homepage at route `/`

Reply:
sounds good. How do you feel about having something like this:
```ts
// test/Test.tsx
export const Test = () => {
  return <></>
}

// index.ts
import {Test} from './Test'

export default Test

```

The reason behind this is I think it is a bit cleaner - index.ts (to me) should be just for re-routing exports. It also allows faster fuzzy searching too

Reply:
Yeah I agree def. Think it's juuust outside scope though so I created this issue: https://github.com/jbx-protocol/juice-interface/issues/1331

Thanks for raising this 🙏